### PR TITLE
fix: 0005481 mssql text for symmetric tables to varchar

### PR DESCRIPTION
**Pull request by SndR85**
Fixes https://www.symmetricds.org/issues/view.php?id=5481. With this change the SymmetricDS tables can be created on MS SQL with _UTF8 of _SC collation by setting the `mssql.use.varchar.for.lob.in.sync` to `true`

### CODE REVIEW

File: symmetric-client/src/main/java/org/jumpmind/symmetric/db/mssql/MsSqlSymmetricDialect.java [hunk 1]
```diff
@@ -73,11 +74,20 @@ public MsSqlSymmetricDialect(IParameterService parameterService, IDatabasePlatfo
     public Database readSymmetricSchemaFromXml() {
         Database db = super.readSymmetricSchemaFromXml();
         if (parameterService.is(ParameterConstants.MSSQL_USE_NTYPES_FOR_SYNC)) {
-            Table table = db.findTable(TableConstants.getTableName(getTablePrefix(),
+            Table sym_table = db.findTable(TableConstants.getTableName(getTablePrefix(),
```
Comment (erilong) [inline symmetric-client/src/main/java/org/jumpmind/symmetric/db/mssql/MsSqlSymmetricDialect.java]: This should be symTable to follow our lower camel case naming convention.

File: symmetric-core/src/main/resources/symmetric-default.properties [hunk 2]
```diff
@@ -3155,7 +3163,7 @@ cloud.bulk.row.terminator=\r\n
 #
 # DatabaseOverridable: true
 # Tags: cloud, snowflake, azure
-cloud.bulk.field.quote=�
```
Comment (erilong) [inline symmetric-core/src/main/resources/symmetric-default.properties]: Why was the cloud.bulk.field.quote parameter changed?  That seems unrelated to the issue.

**Review by erilong — CHANGES_REQUESTED** [review]

File: symmetric-core/src/main/resources/symmetric-default.properties (hunk 2 above)
Comment (SndR85) [reply symmetric-core/src/main/resources/symmetric-default.properties]: Yes, that should not happen, sorry for that, I will try to undo that.

Comment (SndR85) [reply symmetric-core/src/main/resources/symmetric-default.properties]: I have removed this line change.

File: symmetric-client/src/main/java/org/jumpmind/symmetric/db/mssql/MsSqlSymmetricDialect.java (hunk 1 above)
Comment (SndR85) [reply symmetric-client/src/main/java/org/jumpmind/symmetric/db/mssql/MsSqlSymmetricDialect.java]: I have changed it in https://github.com/JumpMind/symmetric-ds/pull/185/commits/69760f23a9004b20e29ec14d17ded246948b6335
